card/mod/standard/set/abstract/wql_search.rb

```diff
@@ -22,9 +22,8 @@ def fetch_query args={}
 end
 
 def standardized_query_args args
-  args.symbolize_keys!
   args[:context] ||= cardname
-  args
+  args.symbolize_keys
```

Comment:
That change means that `args["context"]` will be overridden with cardname. Are you sure that we want that?